src/app/components/Curation/getComponentName/index.ts

```diff
@@ -11,6 +11,7 @@ export const COMPONENT_NAMES = {
   NOT_SUPPORTED: 'not-supported',
   MOST_READ: 'most-read',
   RADIO_SCHEDULE: 'radio-schedule',
+  EMBED: 'vj-embed',
```

Comment:
Do we need to include `vj` in here? As there might be other types of Embed in future (not just VJ ones)

```suggestion
  EMBED: 'embed',
```